src/content/docs/apm/apm-ui-pages/events/record-deployments.mdx

```diff
@@ -321,23 +321,21 @@ Some agents have additional methods to record deployments:
 * **Python**: Use the [`record-deploy`](/docs/agents/python-agent/installation-configuration/python-agent-admin-script#record-deploy) subcommand of the `newrelic-admin` script.
 * **Ruby**: Use a [Capistrano recipe](/docs/agents/ruby-agent/features/recording-deployments-ruby-agent#capistrano3).
 
-## Notify your team of deployments [#webhooks]
+## Notify your team about deployments [#webhooks]
 
-After a deployment is recorded using the REST API, you can optionally notify a webhook endpoint of the deployment.
+After a deployment is recorded, through either the REST API or the newer [GraphQL API](/docs/change-tracking/change-tracking-introduction), you can notify your team members using a webhook.
 
-The destination of the webhook can be your Slack instance. To use webhooks to set up a deployment notification for a Slack channel:
+You can send deployment data to a variety of webhook destinations. Simply follow the instructions for your whatever tool you're using&mdash;except Slack. If you want to use a Slack webhook, you must use the legacy New Relic Alerts app instead of the New Relic Slack app like this:
 
-1. Log in to your Slack account as an admin, then go to **App directory > Manage > Apps**.
-2. Search for your New Relic app, then select **Add configuration**.
-3. From **Post to channel**, select an existing Slack channel or add a new channel, then **Add configuration**.
-4. From the list of options, copy the webhook URL.
-5. Go to **[one.newrelic.com](https://one.newrelic.com/all-capabilities) > ([user menu](/docs/accounts/accounts-billing/general-account-settings/intro-account-settings)) > Account settings > Integrations > Deploy notifications > Webhook**.
-6. Paste the Slack webhook URL, then save.
-7. Optional: Send a test message.
-
-<Callout variant="tip">
-  You can also use webhooks, Slack channels, and other options for [alert notifications](/docs/alerts/new-relic-alerts/managing-notification-channels/notification-channels-control-where-send-alerts).
-</Callout>
+1. Log in to your Slack account as an admin, and then go to **Apps**.
+2. Search for **New Relic Alerts**, and when you see that tile, click on it.
```

Comment:
I feel like this sentence feels a little awkward and clunky-- what if it read:

"Search for **New Relic Alerts**, and click on that tile when it appears."

Reply:
@homelessbirds thanks for the suggestion! I'll use it!